src/cache.js

```diff
@@ -41,7 +46,7 @@ export default function cacheMw(settings = {}) {
 
         const cacheKey = createCacheKey(resourceId, method, params);
 
-        const cachedResult = await cache.get(bin, cacheKey);
+        const cachedResult = await cache.get(bin, cacheKey).catch(errorHandler);
```

Comment:
I think you should make errorHandler configurable. Maybe someone want to send errors of that type to sentry / whatever.

Reply:
👍